dspace-api/src/main/java/org/dspace/importer/external/metadatamapping/contributor/SplitMetadataContributor.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.importer.external.metadatamapping.contributor;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+import org.dspace.importer.external.metadatamapping.MetadataFieldMapping;
+import org.dspace.importer.external.metadatamapping.MetadatumDTO;
+
+public class SplitMetadataContributor<T> implements MetadataContributor<T> {
+
+    private final MetadataContributor<T> innerContributor;
+    private String regex;
+
+    public SplitMetadataContributor(MetadataContributor<T> innerContributor, String regex) {
```

Comment:
Please also add JavaDocs to this constructor method, especially to describe the "regex" and how it is used (obviously this regex is used to split the values, but we should describe that in JavaDocs)